powered-by.md

```diff
@@ -170,9 +160,8 @@ across all screens
   - PanTera is a tool for exploring large datasets. It uses Spark to create XY and geographic 
   scatterplots from millions to billions of datapoints.
   - Components we are using: Spark Core (Scala API), Spark SQL, and GraphX
-- <a href="http://www.peerialism.com">Peerialism</a>
 - <a href="http://www.planbmedia.com">PlanBMedia</a>
-- <a href="http://prediction.io/">PredicitionIo</a>
+- <a href="http://predictionio.apache.org/index.html/">Apache PredicitionIo</a>
```

Comment:
`Apache PredictionIO`

Reply:
There may not be much point, yeah, in SNAPSHOT artifacts. I don't know how to shut them off -- @shaneknapp is that just a Jenkins job? The argument for turning it off is that it has been somewhat controversial in the past to provide any binaries from the project that aren't PMC releases. It's allowed if heavily signposted. But, maybe little point in dumping 1GB of artifacts per night into the repo.

Reply:
we are currently not publishing any snapshot builds except for docs...  i can easily shut these builds off.

Reply:
The docs build probably isn't useful and just filling up dist, though that's not what I was referencing here. It's: https://repository.apache.org/snapshots/org/apache/spark and while that seems to down now, I did see recent snapshots of the artifacts. However @gatorsmile and others note that they may be useful, so let's keep whatever it is.

Reply:
kk